src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig

```diff
@@ -105,7 +105,7 @@
 
 {% macro table_header_tools(form_add, form_remove, can_add) %}
     <button
-            data-udw-config="{{ ez_udw_config('single', {}) }}"
+            data-udw-config="{{ ez_udw_config('add_location', {}) }}"
```

Comment:
I'm not sure about this one. @Nattfarinn correct me if I'm wrong but changing config name here is a BC break in case someone already used custom configuration for `single`.

Reply:
There is a tiny risk of bahavioral BC here, but I wouldn't bother much with it.